04/User.js

```diff
@@ -1,3 +1,46 @@
 export default class User {
+    constructor({
+        email,
+        password
+    }) {
+        this.email = this._checkEmail(email);
+        this.password = this._checkPassword(password);
+    }
+
+    getEmail() {
+        return this.email;
+    }
+
+    getPassword() {
+        return this.password;
+    }
+
+    login() {
+        if (this.email.includes('devmentor.pl')) {
+            return true;
+        } else return false;
+    }
+
+    _checkEmail(email) {
+        if (/\S+@\S+\.\S+/.test(email)) {
+            return email;
+        } else {
+            throw new Error('Invalid email address')
+        }
+    }
+
+    _checkPassword(password) {
+        const PASSWORD_MIN_LENGHT = 7
```

Comment:
Jak już chcesz iść w tą stronę to myślę, że spokojnie możesz wrzucić tą zmienną powyżej klasy. Będzie to czytelniejsze rozwiązanie :)